src/openapi/transport/retry.spec.js

```diff
@@ -55,15 +55,15 @@ describe('openapi TransportRetry', () => {
         });
         setTimeout(() => {
             expect(transport.delete.mock.calls.length).toEqual(1);
-            transport.deleteReject(new TypeError('First Network request failed'));
+            transport.deleteReject({ status: 503, response: 'first' });

```

Comment:
Could you explain why we changed this test to status: 503 from TypeError?

Reply:
These type errors don't have a status or a isNetworkError - so they aren;t things we should really be retrying..
I've changed the logic so we only retry if there is a status and we match it.
Previously if the error did not have a status it would always retry.. thats highly dangerous.